base_station/gui/src/components/LCMEcho.vue

```diff
@@ -175,6 +176,7 @@
           {'topic': '/target_bearing', 'type': 'TargetBearing'},
           {'topic': '/target_list', 'type': 'TargetList'},
           {'topic': '/target_orientation', 'type': 'TargetOrientation'},
+          {'topic': '/teleop_en', 'type': 'Enable'},
```

Comment:
should ```{'topic': '/auton', 'type': 'AutonState'}``` be auton_en and Enable?

Reply:
This^ but also, can we just write out /auton_enabled and /teleop_enabled?

Reply:
My bad I didn't see this until now, will make changes later tonight or tomorrow morning